test/src/main/java/io/strimzi/test/TestUtils.java

```diff
@@ -133,7 +133,7 @@ public static long waitFor(String description, long pollIntervalMs, long timeout
                 result = ready.getAsBoolean();
             } catch (Exception e) {
                 exceptionMessage = e.getMessage();
-                if (++exceptionCount == 1) {
+                if (++exceptionCount == 1 && exceptionMessage != null) {
```

Comment:
I think you should print a warning even if the `exceptionMessage` is null and print `exceptionMessage` only if it's filled.